src/axom/klee/Geometry.hpp

```diff
@@ -69,21 +78,169 @@ class Geometry
            std::shared_ptr<GeometryOperator const> operator_);
 
   /**
-   * Get the format in which the geometry is specified.
+   * Create a Geometry object based on a blueprint tetrahedral mesh.
+   *
+   * \param startProperties the transformable properties before any
+   * operators are applied
+   * \param meshGroup a simplex geometry in blueprint format.
+   *   The elements should be segments, triangles or tetrahedra.
+   * \param topology The blueprint topology to use.
+   * \param operator_ a possibly null operator to apply to the geometry.
+   *
+   * \internal TODO: Is this the simplex requirement overly restrictive?
```

Comment:
```suggestion
   * \internal TODO: Is the simplex requirement overly restrictive?
```

Reply:
Deleted the note.  It was just for me to think about.